Feature 52 mode brightnesstemperature

CPKalb opened the pull request:
## Pull Request Testing ##

- [x] Describe testing already performed for these changes:</br>
Checked the online documentation to make sure that the new topic and video are available

- [x] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
Check the branch documentation to be sure it now has a topic number 6, Use Case: MODE Brightness Temperature verification under the METplus Training topics.  When clicking on the topic, it should have an embedded video and text script.  The branch documentation is here: https://metplus-training.readthedocs.io/en/feature_52_mode_brightnesstemperature/modules/METplus/index.html

- [x] Please complete this pull request review by **[Fill in date]**.</br>
As soon as possible

## Pull Request Checklist ##
See the [METplus Workflow](https://dtcenter.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [x] Review the source issue metadata (required labels, projects, and milestone).
- [x] Complete the PR definition above.
- [x] Ensure the PR title matches the feature or bugfix branch name.
- [x] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**
Select: **Organization** level software support **Project** or **Repository** level development cycle **Project**
Select: **Milestone** as the version that will include these changes
- [x] After submitting the PR, select **Linked issues** with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

jprestop reviewed (approved):
Hi Tina.  I corrected a few typos, added a few commas, and added some links to the MET User's Guide and the METplus User's Guide.  I would have some suggestions for changes, like to make certain actions code blocks. For example, for things like "Type vim parm/use_cases/model_applications/convection_allowing_models" so that the users could follow along and copy and paste more easily.  However, I see that the other use case scripts are not like that currently and that your script is very consistent with the others (thank you for  the consistency - that is always appreciated).  So, given that, I would not suggest that you make those changes to this script.  All tests have passed and I approve this PR.

CPKalb commented:
Hi Julie:

Ah, yes.  Thanks.  I tried to keep things consistent above all else.  But I
agree with you that those things should be updated so it is easier to
follow along.

Tina

On Fri, Mar 4, 2022 at 11:47 AM jprestop ***@***.***> wrote:

> ***@***.**** approved this pull request.
>
> Hi Tina. I corrected a few typos, added a few commas, and added some links
> to the MET User's Guide and the METplus User's Guide. I would have some
> suggestions for changes, like to make certain actions code blocks. For
> example, for things like "Type vim
> parm/use_cases/model_applications/convection_allowing_models" so that the
> users could follow along and copy and paste more easily. However, I see
> that the other use case scripts are not like that currently and that your
> script is very consistent with the others (thank you for the consistency -
> that is always appreciated). So, given that, I would not suggest that you
> make those changes to this script. All tests have passed and I approve this
> PR.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dtcenter/METplus-Training/pull/53#pullrequestreview-900586445>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFJHD4U22TN642VK4DNK5EDU6JLEXANCNFSM5P6ESM2A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*************************************************
Christina Kalb
Associate Scientist
National Center for Atmospheric Research
3450 Mitchell Lane
Boulder, CO 80301
***@***.***
303-497-2819
*************************************************

My working day may not be your working day. Please do not feel obliged to
reply to this email outside of your normal working hours.
